examples/light-js/index.html

```diff
@@ -91,12 +100,26 @@ <h2>Remote Peers</h2>
       const ContentTopic = "/js-waku-examples/1/chat/utf8";
       const decoder = createDecoder(ContentTopic);
       const encoder = createEncoder({ contentTopic: ContentTopic });
-      let messages = [];
+      // Each key in messages should be string hash. each value is an obj of class Message
+      let messages = {};
+      class Message {
```

Comment:
seems cumbersome - you can just store object `{ text, timestamp }` and make a string during rendering

Reply:
Removed `class Message` and replaced with a regular object